packages/toolpad-app/src/toolpad/Apps/index.tsx

```diff
@@ -160,11 +162,9 @@ function CreateAppDialog({ onClose, ...props }: CreateAppDialogProps) {
             });
           }
 
-          const appName = config.isDemo ? `demo_app_${Date.now()}` : name;
```

Comment:
Same here I guess this part was accidentally removed? Let me know if it's easier if I fix this myself.